dashboard/agent.py

```diff
@@ -69,7 +76,7 @@ def _load_modules(self):
             c = cls(self)
             dashboard_utils.ClassMethodRouteTable.bind(c)
             modules.append(c)
-        logger.info("Loaded {} modules.".format(len(modules)))
```

Comment:
The best formatter for logger is `%`, because we may set different log levels, and `%` formatter will avoid formatting skipped levels. For example, we set log level to `ERROR`, then
```python
logger.info("Loaded %d modules.", len(modules))  # Skip format.
logger.info("Loaded {} modules.".format(len(modules)))  # Always format.
```

Reply:
That's a good point that I hadn't thought of. Thanks!